ReactiveTwitterSearch/ViewModel/TwitterSearchViewModel.swift

```diff
@@ -50,7 +50,7 @@ class TwitterSearchViewModel {
         }
 
     timer(Constants.TickIntervalLength, onScheduler: QueueScheduler.mainQueueScheduler).startWithNext { [weak self] _ in
-      self?.tweets.value.forEach { $0.updateTime() }
+        self?.tweets.value.update()
```

Comment:
I'm not so keen on this - I think I prefer the original `forEach` based implementation.

I'm all for making good use of protocols extensions, but in this case I don't feel it adds sufficient value.